source/documentation/administration_guide/chap-Renewing_certificates.adoc

```diff
@@ -7,28 +7,42 @@
 
 [WARNING]
 ====
-Do _NOT_ let certificates expire. If they expire, the environment becomes non-responsive and recovery is an error prone and time consuming process. If they do expire, you can manually renew them by following the instructions in link:https://access.redhat.com/solutions/3532921[How to manually renew RHV host SSL certificate if expired?]. Once you have completed the steps to manually renew the certificates, follow the steps in this procedure to properly enroll {virt-product-shortname} host certificates.
+Do not let certificates expire. If they expire, the host and {hypervisor-shortname} stop responding, and recovery is an error-prone and time-consuming process. If they do expire, you can manually renew them by following the instructions in link:https://access.redhat.com/solutions/3532921[How to manually renew RHV host SSL certificate if expired?]. Once you have completed the steps to manually renew the certificates, follow the steps in this procedure to properly enroll {virt-product-shortname} host certificates.
 ====
 
 .Procedure
 
-. In the Administration Portal, click menu:Compute[Hosts].
-. Migrate the virtual machines away from the host.
-. Click menu:Management[Maintenance] and btn:[OK].
-. Once the host is in maintenance mode, click menu:Installation[Enroll Certificate].
-. Once enrollment is complete, click menu:Management[Activate].
+. Renew the host certificates:
+.. In the Administration Portal, click menu:Compute[Hosts].
+.. Click menu:Management[Maintenance] and then click btn:[OK].
+The virtual machines should automatically migrate away from the host. If they are pinned or otherwise non-migratable, you need to shut them down.
+.. Once the host is in maintenance mode, and there are no more virtual machines remaining on this host, click menu:Installation[Enroll Certificate].
+.. Once enrollment is complete, click menu:Management[Activate].
 
-.Next steps
-You must also renew Engine certificates.
-
-. Run `engine-setup` from an SSH or console session on the {engine-appliance-name} system. The `engine-setup` script prompts you with some configuration questions.
+. Renew the {engine-name} certificates:
+.. For a self-hosted engine, log in to the host and put it in global maintenance mode:
```

Comment:
It might be easier to read if you changed "For a self-hosted engine, " to "Self-hosted engine only:" or "Self-hosted engine:". Fewer words are easier to scan. 

Same suggestion for the step for activating the host.